ccd-definition/CaseField/CaseField-CUI-R2-nonprod.json

```diff
@@ -33,6 +33,40 @@
     "Searchable": "N",
     "Publish": "Y"
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "helpWithFeesReferenceNumber",
+    "Label": " ",
+    "FieldType": "HelpWithFees",
+    "SecurityClassification": "Public",
+    "Searchable": "N",
+    "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "CIVIL",
+    "ID": "caseTitleWithHwFReference",
+    "Label": "# ${applicant1.partyName} Vs ${respondent1.partyName}",
+    "FieldType": "Label",
+    "SecurityClassification": "Public",
+    "Searchable": "N"
```

Comment:
Move these to HWF related file CaseField-HWF-R2-CUI-nonprod.json. Also please check others PR how these are define and follow same to make these consistent